pkg/cloudprovider/providers/azure/azure_standard.go

```diff
@@ -244,11 +244,11 @@ func getServiceName(service *v1.Service) string {
 
 // This returns a prefix for loadbalancer/security rules.
 func getRulePrefix(service *v1.Service) string {
-	return cloudprovider.GetLoadBalancerName(service)
```

Comment:
We'll want someone from azure to look at this but this should probably be:
```go
func (az *Cloud) getRulePrefix(service *v1.Service) string {
    return az.GetLoadBalancerName(...)
}
```

This is because we want `cloudprovider.DefaultGetLoadBalancerName` originating from one place so it's easier to change in the future.

Reply:
Same applies for the methods below

Reply:
cc @kubernetes/sig-azure-misc

Reply:
Agreed. Please use az.GetLoadBalancerName() instead

Reply:
Should I update every methods parameter list, that sets the loadbalancername within its method, to include `ctx context.Context` and `clusterName string` ? @feiskyer @andrewsykim

Reply:
@feiskyer would be good if you can comment on this. IMO I'd say ignore `ctx` and `clusterName` for now since it's not used for LB names anyways. When azure devs get to it they can add the clusterName and ctx to those methods.

Reply:
i.e. `clusterName: ""` and `context.TODO()`

Reply:
Let's ignore clusterName and ctx for now. 

Please notice that the name here is only used for frontend configuration name. There is actually another `getLoadBalancerName()` for getting Azure load balancer name. It may be renamed to `getAzureLoadBalancerName()`.